### ISSUE TITLE
Ruleset parsing fails with an Invalid Datetime error on what appears to be a valid rule

### ISSUE THREAD

**Issue by aggregat4** [fmeringdal/rust_rrule#2]
I have been trying to integrate rust_rrule in a tool where I deal with ical calendars. I have been extracting DTSTART/RRULE combinations from an Outlook ical feed and letting rust_rrule parse them. 

Using v0.3.3.

It fails with parsing strings like the following:

```
#[test]
    fn rruleset_parsing() {
        "DTSTART;VALUE=DATE:20200812\nRRULE:FREQ=WEEKLY;UNTIL=20210511T220000Z;INTERVAL=1;BYDAY=WE;WKST=MO".parse::<RRuleSet>().unwrap();
    }
```

Here is the error message:

```
thread 'meeters_ical::tests::rruleset_parsing' panicked at 'called `Result::unwrap()` on an `Err` value: RRuleParseError("Invalid datetime: DTSTART;VALUE=DATE:20200812\nRRULE:FREQ=WEEKLY;UNTIL=20210511T220000Z;INTERVAL=1;BYDAY=WE;WKST=MO")', src/meeters_ical.rs:296:129
```

**Comment by fmeringdal** [comment]
Thanks for reporting the issue! 
It was indeed a bug when the rrule string contains a date value and not a datetime value. This should now have been fixed in version 0.3.4, which is able to correctly parse the given rrule string. Please let me know if the issue still persists.

**Comment by aggregat4** [comment]
Yes, this fixes the isse. Thank you!

I have however uncovered the next problem, but will make a separate issue for that.
